### PR TITLE
Update MERCHANT_INTEGRATION.md

### PR DESCRIPTION
**Pull request by vunderkind**
There was a diff in the original SPL code that may be confusing for people who just want to copy the snippet and test out the SPL token transfer. I cleaned it up so it reads properly in JS.

### CODE REVIEW

**Comment by jordansexton** [comment]
@cogoo would you review this?

File: docs/src/core/MERCHANT_INTEGRATION.md [hunk 1]
```diff
@@ -132,7 +130,7 @@ For SPL Token transfers, use the `spl-token` parameter. The `spl-token` is the m
     const label = 'Jungle Cats store';
     const message = 'Jungle Cats store - your order - #001234';
     const memo = 'JC#4098';
-+   const splToken = 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v';
+    const splToken = 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v';
```
Comment (jordansexton) [inline docs/src/core/MERCHANT_INTEGRATION.md:133]: This needs to be a pubkey, see https://github.com/solana-labs/solana-pay/pull/49

**Review by jordansexton — CHANGES_REQUESTED** [review]
Thanks! If you merge from master into here, it should fix the one issue I commented on.

**Comment by vunderkind** [comment]
Thanks everyone! 👍🏿

**Pull request merged**